src/pages/Home/components/Portfolio/Prompt/Prompt.tsx

```diff
@@ -360,8 +362,11 @@ export function Prompt() {
           // with the arguments specified in the function call and
           // let it call the hypothetical API.
           try {
+            capture('CoreAssistantFunctionCall', {
+              functionName: call.name,
+              userMessage: message,
+            });
```

Comment:
Shouldn't we encrypt these events? The message may contain sensitive data.
```suggestion
            capture('CoreAssistantFunctionCall', {
              functionName: call.name,
              userMessage: message,
            }, false, true);
```